robot/Cumulus/resources/NPSP.py

```diff
@@ -967,4 +971,17 @@ def verify_expected_values(self,ns_ind,obj_api,rec_id,**kwargs):
             table=obj_api
        rec=self.salesforce.salesforce_get(table,rec_id)
        for key, value in kwargs.items():
-           self.builtin.should_be_equal_as_strings(rec[key], value)   
\ No newline at end of file
+           self.builtin.should_be_equal_as_strings(rec[key], value)
+
+    def batch_data_import(self):
```

Comment:
Every Robot Framework keyword should have a docstring so that it has a meaningful description in the generated docs.

Reply:
Makes sense. Fixed in 8178a46e1